target/linux/mediatek/image/Makefile

```diff
@@ -13,6 +13,10 @@ loadaddr-$(CONFIG_TARGET_mediatek_mt7623) := 0x80008000
 loadaddr-$(CONFIG_TARGET_mediatek_mt7629) := 0x40008000
 loadaddr-$(CONFIG_TARGET_mediatek_filogic) := 0x48000000
 
+define Build/zyimage
+	$(STAGING_DIR_HOST)/bin/zyimage $(1) $@
+endef
```

Comment:
Instead of adding ` Build/zyimage` here again move the definition from `target/linux/ramips/image/Makefile` to `include/image-commands.mk` and use the same definition for the ramips and the mediatek target.